geode-docs/managing/security/implementing_authorization.html.md.erb

```diff
@@ -56,13 +56,23 @@ which classifies whether the operation as
 The operations are not hierarchical;
 `MANAGE` does not imply `WRITE`, and `WRITE` does not imply `READ`.
 
-Some operations further specify a region name in the permission.
+Some `DATA` operations further specify a region name in the permission.
 This permits restricting operations on that region to only those
 authorized principals.
 And within a region, some operations may specify a key.
 This permits restricting operations on that key within that region to 
 only those authorized principals.
 
+Some `CLUSTER` operations further specify a finer-grained
+target for the operation.
+Specify the target with a string value of:
+
+- `DISK` to target operations that write to a disk store
+- `GATEWAY` to target operations that manage gateway senders and receivers
+- `QUERY` to target operations that manage both indexes and continuous
+ queries
+- `JAR` to target operations that deploy code to servers
+
```

Comment:
Will add the LUCENE target before I merge this PR. Thanks!

I created https://issues.apache.org/jira/browse/GEODE-3324 to complete the changes specified in the wiki proposal.